File: commonLib/src/main/java/com/travel/commonlib/packages/BaggageList.java
package com.travel.commonlib.packages;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class BaggageList implements Serializable {
    @SerializedName("Text")
    @Expose
    private String text;
    @SerializedName("Value")
    @Expose
    private String value;
    @SerializedName("Selected")
    @Expose
    private boolean selected;
    @SerializedName("Price")
    @Expose
    private double price;

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public boolean isSelected() {
        return selected;
    }

    public void setSelected(boolean selected) {
        this.selected = selected;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }
}

File: commonLib/src/main/java/com/travel/commonlib/flight/PaxModel/BaggageList.kt
package com.travel.commonlib.flight.PaxModel

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class BaggageList : Serializable {
    @SerializedName("Text")
    @Expose
    var text: String? = null

    @SerializedName("Value")
    @Expose
    var value: String? = null

    @SerializedName("Selected")
    @Expose
    var isSelected = false

    @SerializedName("Price")
    @Expose
    var price = 0.0

    @SerializedName("Currency")
    @Expose
    var currency: String? = null

    @SerializedName("Weight")
    @Expose
    var weight = 0

    @SerializedName("Type")
    @Expose
    var type: String? = null

    @SerializedName("BookingTravelerRef")
    @Expose
    var bookingTravelerRef: Any? = null

    @SerializedName("NumberOfPieces")
    @Expose
    var numberOfPieces = 0
}